hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/SCMCommonPlacementPolicy.java

```diff
@@ -196,7 +200,7 @@ public boolean hasEnoughSpace(DatanodeDetails datanodeDetails,
 
     for (MetadataStorageReportProto reportProto
         : datanodeInfo.getMetadataStorageReports()) {
-      if (reportProto.getRemaining() > metaSizeRequired) {
+      if (reportProto.getRemaining() > metadataSizeRequired) {
```

Comment:
Can you add a “metadataSizeRequired not 0" if check around this for loop?  since metadataSizeRequired is passed as 0 in container replication.

Reply:
Thanks @ChenSammi I added the check and addressed some minor test failures.